spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -186,14 +186,14 @@ def filtered_hand_rgb(self):
 
     def detections_cb(self, msg):
         timestamp, detections_str = msg.data.split("|")
-        self.detections_buffer["detections"][int(timestamp)] = detections_str
+        self.detections_buffer["detections"][str(timestamp)] = detections_str
```

Comment:
Double check this is correct!

Reply:
This correction was done by Kavit, I copied his change. We had diagnosed that if timestamp is converted to int or only if its .nsec part is used then we don't get timestamp that matches with the image publisher